src/Image.cc

```diff
@@ -420,6 +428,24 @@ Image::loadSurface() {
   if (isJPEG(buf)) return loadJPEG(stream);
 #endif
 
+// svg
+#ifdef HAVE_RSVG
+  // confirm svg using first 1000 chars
+  // if a very long comment precedes the root <svg> tag, isSVG returns false
+  uint8_t head[1000] = {0};
+  fseek(stream, 0 , SEEK_END);
```

Comment:
Thanks @petarov.

I quickly surveyed SVGs in the wild and found only 12% starting with `<svg>` (not sure if my sample set was biased though). Other starts include some combination of `<?xml>`, `<!DOCTYPE>`, `<!-- -->`, and whitespace. For instance Inkscape SVGs start with this preamble:

```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!-- Created with Inkscape (http://www.inkscape.org/) -->
```

What do you think? Should we first try the five bytes before trying the first 1000 since they're around or go straight to the 1000? Like you said, it's only a few ops, but since these lines may get a lot of use, I'm happiest doing it best. Please tell me either way.